Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst

```diff
@@ -0,0 +1,107 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_mutual_authentication:
+
+*********************
+Mutual Authentication
+*********************
+
+.. include:: ../../../beta.rst
+
+Mutual Authentication and mTLS background
+#########################################
+
+Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity, and confidentiality of data exchanged between two entities over a network.
+
+Unlike traditional TLS, which involves a one-way authentication process where the client verifies the server's identity, mutual TLS adds an additional layer of security by requiring both the client and the server to authenticate each other.
+
+Mutual TLS aims at providing authentication, confidentiality and integrity to service-to-service communications. 
+
+Mutual Authentication in Cilium
+###############################
+
+Cilium’s mTLS-based Mutual Authentication support brings the mutual authentication handshake out-of-band for regular connections.
```

Comment:
`Cilium’s` -> Please avoid typographic (curly) apostrophes/quotes in the source, they can cause all kind of troubles when we try to grep for an expression or process text.

You likely got this one either because you copied the text from somewhere else, or because you wrote it first in a Google Doc where auto-correction turns straight apostrophes into curly ones.